src/install/bin.zig

```diff
@@ -458,7 +634,15 @@ pub const Bin = extern struct {
                     from_remain[0] = 0;
                     const dest_path: [:0]u8 = target_buf[0 .. @intFromPtr(from_remain.ptr) - @intFromPtr(&target_buf) :0];
 
-                    this.setSimlinkAndPermissions(target_path, dest_path);
+                    if (comptime Environment.isWindows) {
+                        var dest_path_buf: bun.PathBuffer = undefined;
+                        @memcpy(dest_path_buf[0..dest_path.len], dest_path);
+                        this.createCmdFile(target_path, &dest_path_buf, dest_path);
+                        this.createPs1File(target_path, &dest_path_buf, dest_path);
+                        this.createShFile(target_path, dest_path);
```

Comment:
i guess its okay if the answer is yes but do we really need all three of these.

(also note: powershell is VERY slow at startup time, like an order of magnitude more so)

Reply:
copying npm/pnpm behavior. By default all are created, but should look into removing these with bun shell